src/main/viz/GenomeTrack.js

```diff
@@ -53,7 +53,7 @@ function renderGenome(ctx: DataCanvasRenderingContext2D,
         // We only push objects in the text case as it involves creating a
         // new object & can become a performance issue.
         // 0.5 = centered
-        ctx.fillText(letter, scale(1 + 0.5 + pos), height - 1);
+        ctx.fillText(letter, scale(1 + 0.5 + pos), height / 2);
```

Comment:
also I don't think passing the center would cause the text to be vertically centered. The alignment of the text should depend on the combination of a few things: http://www.w3schools.com/tags/canvas_textalign.asp

Or am I missing something here?